### ISSUE TITLE
Twitch Notification

### ISSUE THREAD

**Issue by newtox** [zekroTJA/shinpuru#387]
### Type

Enhancement of an existing feature

### Description

Add an optional option so the bot can send a custom message before the embed when sending the notification.

**Comment by itsDkiller** [comment]
What use is this supposed to have?

**Comment by itsDkiller** [comment]
Then wouldn't it be better to mention that role in the given message that shinpuru sends?
This could be selectable to do (or not) in the webinterface.

**Comment by newtox** [comment]
That'd be a good idea, though the mention wouldn't work in an embed right?

**Comment by zekroTJA** [comment]
> That'd be a good idea, though the mention wouldn't work in an embed right?

Yeah, but the mention can be added to the actual message text "before" the embed. Generally, I think that's a good idea which should be relatively easy to implement. The option to mention roles could be added on setting up the twitch notify. 

While doing so, the process of setting up twitch notification could also be re-implemented using message components and embeds for an easier and more user friendly setup.
